.github/workflows/release-dev.yml

```diff
@@ -0,0 +1,19 @@
+# This workflow releases the current package to a dedicated private CodeArtifact repository.
+# One repository may publish more than one package. For more details refer to the release-package Action.
+name: Release to dev branch
+
+on:
+  push:
+    branches:
+      - 'dev-v3-*'
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
are these permissions needed here?

Reply:
Good question. Probably not because the same are defined in the release action we use in the job. I will test

Reply:
Looks like the permissions declaration must be explicit, here is what I got:

```
[Invalid workflow file: .github/workflows/release-dev.yml#L11](https://github.com/cloudscape-design/components/actions/runs/9284358611/workflow)
The workflow is not valid. .github/workflows/release-dev.yml (Line: 11, Col: 3): Error calling workflow 'cloudscape-design/actions/.github/workflows/release.yml@main'. The workflow is requesting 'id-token: write', but is only allowed 'id-token: none'.
```

Reply:
`contents` is used for release notes AFAIK

Reply:
Without contents permission I am getting this:
```
[Invalid workflow file: .github/workflows/release-dev.yml#L14](https://github.com/cloudscape-design/components/actions/runs/9300736117/workflow)
The workflow is not valid. .github/workflows/release-dev.yml (Line: 14, Col: 3): Error calling workflow 'cloudscape-design/actions/.github/workflows/release.yml@main'. The workflow is requesting 'contents: read', but is only allowed 'contents: none'.
```